docs/src/pages/[platform]/connected-components/liveness/index.page.mdx

```diff
@@ -29,7 +29,12 @@ export async function getStaticProps() {
   return { props: {} }
 }
 
-Amplify UI FaceLivenessDetector is powered by [Amazon Rekognition Face Liveness](https://aws.amazon.com/rekognition/). The following screenshots shows an example of FaceLivenessDetector in action.
+The Amplify UI FaceLivenessDetector is powered by [Amazon Rekognition Face Liveness](https://aws.amazon.com/rekognition/face-liveness/). The FaceLivenessDetector supports two types of verification challenges:
+
+ - The **Face Movement and Light Challenge** consists of a series of head movements and colored light tests.
+ - The **Face Movement Challenge** consists only of a series of head movements.
```

Comment:
Added a followup to link to the amplify docs here bc the documentation for the new challenge hasn't been released yet